Mage/src/main/java/mage/abilities/common/SagaAbility.java

```diff
@@ -108,7 +117,10 @@ public SagaChapter getMaxChapter() {
 
     @Override
     public String getRule() {
-        return "<i>(As this Saga enters and after your draw step, add a lore counter."
+        return (readAhead
+                ? "Choose a chapter and start with that many lore counters. " +
+                "Add one after your draw step. Skipped chapters don't trigger."
```

Comment:
Shouldn't this include a "Read ahead" and have the other text formatted as reminder text?

Reply:
good call, fixed